integration/test/API/Brig.hs

```diff
@@ -350,13 +350,16 @@ deleteKeyPackages cid kps = do
   submit "DELETE" $ req & addJSONObject ["key_packages" .= kps]
 
 replaceKeyPackages :: ClientIdentity -> [Ciphersuite] -> [ByteString] -> App Response
-replaceKeyPackages cid suites kps = do
+replaceKeyPackages cid suites kps = replaceKeyPackages' cid (Just suites) kps
+
+replaceKeyPackages' :: ClientIdentity -> Maybe [Ciphersuite] -> [ByteString] -> App Response
```

Comment:
The convention we've been sticking to in cases like this one is to create a data type say `ReplaceKeyPackages` with a `Default` instance and redefine `replaceKeyPackages` as `ClientIdentity -> ReplaceKeyPackages -> App Response`, so we can have something that resembles named arguments. Having primed method names is confusing and doesn't really scale.